rpg.py

```diff
@@ -167,13 +189,20 @@ def run(self):
 
         while not self.is_over():
             acting_player, current_game_time = self.get_from_queue()
+            actions = {'fight': acting_player.act(self.players), 'stats': acting_player.view_stats(self.players)}
             if acting_player.is_alive():
-                updated_players = acting_player.act(self.players)
-                self.players = updated_players
+                if acting_player.team == 1:
+                    action = None
+                    while action is None:
+                        player_input = input('>')
+                        action = actions.get(player_input)
+                        print(action)
```

Comment:
So just remember, `action` is a callable function now. You will want to just do
```
                    while action is None:
                         player_input = input('>')
                         action = actions.get(player_input)
                    # execute the action to updated the players in the game
                   updated_players = action()
```

Reply:
I'm assuming that the method returns a modified list of players.

Reply:
Also note that using `view` will essentially waste the turn. Do know you why that is?

Reply:
I know, Because when You view the stats, you end the Action method and start the act method, the action method, technically called the attack method and therefore stopped the attack method and wasted the turn, I fixed this by stating that I want to keep asking for an input for action until the input calls the act method